mesheryctl/internal/cli/root/registry/generate.go

```diff
@@ -79,14 +79,14 @@ mesheryctl registry generate --registrant-def [path to connection definition] --
 			return ErrUpdateRegistry(err, modelLocation)
 		}
 		utils.Log.SetLevel(logrus.DebugLevel)
-		logFilePath := filepath.Join(logDirPath, "Logs")
+		logFilePath := filepath.Join(logDirPath, "Logs.txt")
```

Comment:
```suggestion
		logFilePath := filepath.Join(logDirPath, "model-generation.log")
```

Are these model-generation or registry-import logs? Either way, let's name accordingly. Please

Reply:
It's it's both, then `registry.log` will work.

Reply:
These are the logs of `Model component Generator`. In the action we only want to mail the `error` so it's better we have 2 separate logs one that shows the error and the other that stores the general logs.

Reply:
Will you add a note here highlighting how thoughtful we have been and how users can take note that a missing `model-generation-errors.log` is a good thing, but that it and the `model-generation.log` files can be referenced? Explain how they are used, please.